webapp/src/Controller/Jury/ClarificationController.php

```diff
@@ -352,7 +352,17 @@ public function changeQueueAction(Request $request, int $clarId): Response
         if ($queue === "") {
             $queue = null;
         }
-        $clarification->setQueue($queue);
+
+        // Either this is the first in a thread (of 1 or more) or this 2nd or more.
+        $firstClarification = $clarification->getInReplyTo();
+        $curClarification = $clarification;
+        if ($firstClarification) {
+            $curClarification = $firstClarification;
+        }
```

Comment:
```suggestion
        $curClarification = $clarification->getInReplyTo() ?? $clarification;
```

Or maybe `?:` if it doesn't return `null` but `false`

Reply:
`public function getInReplyTo(): ?Clarification`

So `??` should be good?